### PR TITLE
Iteration 4 Changes: Instructor Profile

### PR DESCRIPTION
**Pull request by sharvani-ramineni**
Iteration 4 Changes : Instructor Profile

### CODE REVIEW

File: app/views/main/_navbar.html.erb [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
     <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
       <ul class="nav navbar-nav navbar-right">
+      <%= link_to 'My Profile', profile_path, class: 'btn btn-primary' %>
```
Comment (khajifaizanali1999) [inline app/views/main/_navbar.html.erb]: shouldn't be this activated when user is instructor

File: app/helpers/profiles_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module ProfilesHelper
```
Comment (satyabhavsar1) [inline app/helpers/profiles_helper.rb]: Do we need this file?

File: config/routes.rb [hunk 3]
```diff
@@ -52,5 +52,9 @@
   end
   post 'filter_data' => 'challenges#filter_data'
 
+  get '/profile', to: 'profiles#show', as: "profile"
+  get '/edit_profile', to: 'profiles#edit', as: "edit_profile"
```
Comment (satyabhavsar1) [inline config/routes.rb:56]: Do we need both edit and update routes?

Comment (sharvani-ramineni) [reply config/routes.rb:56]: yes one is get and one is post

File: app/helpers/profiles_helper.rb (hunk 2 above)
Comment (sharvani-ramineni) [reply app/helpers/profiles_helper.rb]: not needed, removing

File: app/views/main/_navbar.html.erb (hunk 1 above)
Comment (sharvani-ramineni) [reply app/views/main/_navbar.html.erb]: don't we need profile for trainee as well?

**Review by satyabhavsar1 — APPROVED** [review]

**Pull request merged**